src/parse-templates.ts

```diff
@@ -375,6 +376,9 @@ export function parseTemplates(
 
         results.push({
           type: 'template-tag',
+          // we know that the templateTag has to be set because we have set the templateStart and templateEnd tokens
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Let's just pass it in to `parseTemplateTag`, since we already type narrow there we won't need this assertion / cast